docs/source/using-discovery/workingwithgpu.rst

```diff
@@ -140,21 +135,19 @@ There are several versions of CUDA Toolkits on Discovery, as listed below.::
   cuda/11.3
   cuda/11.4
 
-You can always use the ``module avail`` command to check for the latest software versions on Discovery as well. To see details on a specific CUDA toolkit version, use ``module show``. For example, ``module show cuda/11.4``.
+Use the ``module avail`` command to check for the latest software versions on Discovery. To see details on a specific CUDA toolkit version, use ``module show``. For example, ``module show cuda/11.4``.
 
-To add CUDA to your path use ``module load``. For example, type ``module load cuda/11.4`` to load version 11.4 to your path.
+To add CUDA to your path, use ``module load``. For example, type ``module load cuda/11.4`` to load version 11.4 to your path.
 
 Use the command ``nvidia-smi`` (NVIDIA System Management Interface) inside a GPU node to get the CUDA driver information and monitor the GPU device.
 
 Using GPUs with PyTorch
 ========================
-You should use PyTorch with a conda virtual environment if you need to run the environment on the Nvidia GPUs on Discovery.
-
-The following examples demonstrate how to build PyTorch inside a conda virtual environment for CUDA version 11.7. 
-Make sure that you are on a GPU node before loading the environment and also please note that the installation does not work on k40m or k80 GPU's
+You should use PyTorch with a conda virtual environment if you need to run the environment on the Nvidia GPUs on Discovery. The following examples demonstrate how to build PyTorch inside a conda virtual environment for CUDA version 11.7. 
+Make sure that you are on a GPU node before loading the environment. Please note, the installation does not work on k40m or k80 GPU's
```

Comment:
Should we tell them how to confirm they are on a gpu node? "hostname" or similar command?

Could also be more explicit here like this: Please note, the PyTorch installation does not work on k40m or k80 GPU's.

Reply:
@SerenaCaplins, yes, this has now been modified to ensure that they are not on any of the Kepler nodes. I've expanded on the Notes for this section.